internal/kubernetes/metadata/metadata.go

```diff
@@ -41,11 +45,17 @@ func InitRemoteWorkProcessorMetadata() {
 		log.Fatal(err)
 	}
 
+	image, err := getEnv(IMAGE_ENV_VAR)
+	if err != nil {
+		log.Fatal(err)
```

Comment:
Shouldn't you also return the error, or you can continue without the image?

Reply:
Invocation of `log::Fatal` will panic the program and exit immediately.